pkg/model/label.go

```diff
@@ -39,12 +42,41 @@ func GetArchivePathPrefixToLabels(repo string, prefixes ...string) string {
 	return fmt.Sprint(getArchivePathToLabels(), repo+"/"+strings.Join(prefixes, "/"))
```

Comment:
this comment for more explanations: how are we supposed now to carry on with prefix-based search on labels?

Reply:
I am not sure I understand this comment. The prefix should work as expected. Ref: https://github.com/oneconcern/datamon/blob/labels/cmd/datamon/cmd/cli_test.go#L609

Reply:
ok